### DISCUSSION TITLE
When you select Latest in build, is that Beta?

### DISCUSSION THREAD

**Discussion by tom11011** [codemagic-ci-cd/codemagic-docs#2167] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Choose Latest in Codemagic build

### Expected results

Able to move to Apple review.

### Actual results

Although it works for testflight, Apple says its beta when trying to get reviewed for production.

I thought "Latest" would mean latest production ie 15.0.1 but it seems like latest is 15.1Beta.

**Comment by edeuss** [comment]
@tom11011 Same issue for me. I have tried building by selecting 15.0 and 15.0.1 and Apple says it's a build using the beta also. We did a build yesterday that didn't have this issue.

**Reply by tom11011** [reply]
I built it with 14.3.1 and worked ok.

**Comment by bracket-teknoloji** [comment]
Same here.

**Comment by dtrdic** [comment, marked as answer]
Looks like there is an [ongoing issue](https://developer.apple.com/forums/thread/740844) on Apple side when using the 15.0.1 version. The temporary workaround is to downgrade your Xcode version to 14.3.1

**Comment by mohachouch** [comment]
@dtrdic Two weeks ago, I successfuly  send app with Xcode 15.0, Build version 15A240d.

But with version Xcode 15.0.1 Build version 15A507, it's not working.

Can you please revert Xcode 15.0, Build version 15A240d

Thanks

**Reply by martinjeret** [reply]
Reverting Xcode versions is not possible since they are installed on the host and require updating all hosts in the data center.

**Reply by tom11011** [reply]
You can on codemagic, you choose the virtual machine you want to compile on.

**Reply by mohachouch** [reply]
Thanks, we will pray that Apple corrects quickly

**Comment by mohachouch** [comment]
It's ok, Apple fix the bug, we can now send app with Xcode 15.0.1
